components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.endpoint/src/gen/java/org/wso2/carbon/identity/configuration/mgt/endpoint/SearchApi.java

```diff
@@ -0,0 +1,57 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.wso2.carbon.identity.configuration.mgt.endpoint;
+
+import org.apache.cxf.jaxrs.ext.search.SearchContext;
+import org.wso2.carbon.identity.configuration.mgt.endpoint.dto.ResourcesDTO;
+import org.wso2.carbon.identity.configuration.mgt.endpoint.factories.SearchApiServiceFactory;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Response;
+
+@Path("/search")
+@Consumes({"application/json"})
+@Produces({"application/json"})
+@io.swagger.annotations.Api(value = "/search", description = "the search API")
+public class SearchApi {
+
+    private final SearchApiService delegate = SearchApiServiceFactory.getSearchApi();
+
+    @GET
+
```

Comment:
remove unnecessary new line

Reply:
Intentionally left due to a generated class.